jangaroo-browser/src/main/joo/js/MouseEvent.as

```diff
@@ -0,0 +1,35 @@
+package js {
+
+/**
+ * Events that occur due to the user interacting with a pointing device (such as a mouse). Common events using this
+ * interface include click, dblclick, mouseup, mousedown.
+ */
+[Native("MouseEvent")]
+public class MouseEvent extends UIEvent {
```

Comment:
I think I already did this refactoring on the 4-for-typescript-migration branch. Let's check.

Reply:
Agreed on taking the changes here